README.adoc

```diff
@@ -0,0 +1,40 @@
+[[git-plugin]]
+= Tool Environment Plugin
+:toc: macro
+:toc-title:
+
+link:https://ci.jenkins.io/job/Plugins/job/toolenv-plugin/job/master/[image:https://ci.jenkins.io/job/Plugins/job/toolenv-plugin/job/master/badge/icon[Build Status]]
+link:https://github.com/jenkinsci/toolenv-plugin/graphs/contributors[image:https://img.shields.io/github/contributors/jenkinsci/toolenv-plugin.svg?color=blue[Contributors]]
+link:https://github.com/jenkinsci/toolenv-plugin/releases/latest[image:https://img.shields.io/github/release/jenkinsci/toolenv-plugin.svg?label=changelog[GitHub release]]
```

Comment:
There don't seem to be any releases of the plugin that are recorded as GitHub releases.  There are two tags which could be used to record a GitHub release, but no release has been recorded for those tags.

This svg is a good choice if the plugin will be [GitHub Releases, possibly with release drafter](https://www.jenkins.io/doc/developer/publishing/documentation/#changelogs) to automate the creation of changelogs.  A [video tutorial](https://www.youtube.com/watch?v=PaQsvli92XY) on release drafter in the Jenkins project is also available.
```suggestion
link:https://github.com/jenkinsci/toolenv-plugin/releases/latest[image:https://img.shields.io/github/release/jenkinsci/toolenv-plugin.svg?label=changelog[GitHub release]]
```

Reply:
I'm the maintainer and have already enabled release drafter on this plugin and added the config in #PR5. Hopefully this release will be the first one using it.